dokumentpakking/src/main/java/no/difi/meldingsutveksling/dokumentpakking/service/SBDFactory.java

```diff
@@ -39,25 +39,26 @@ public StandardBusinessDocument createNextMoveSBD(Organisasjonsnummer avsender,
                                                       String process,
                                                       String documentType,
                                                       Object any) {
-        Optional<DocumentType> type = DocumentType.valueOfDocumentType(documentType);
+        Optional<MessageType> type = MessageType.valueOfDocumentType(documentType);
```

Comment:
Kva var tittelen på commit-en igjen? DocumentType -> MessageType?

Reply:
"MOVE-2359: Wrong terminology for DocumentType, was actually MessageType. Added validation for fiksio message types" - hvordan det?

Reply:
Optional of MessageType vert henta vha. valueOfDocumentType. Var berre namngivinga eg stussa litt på.

Reply:
Ja, den er riktig nå. MessageType utledes fra DocumentType.

Reply:
Eg som ikkje kunne lesa.